src/NBitcoin/StandardScripts.cs

```diff
@@ -17,32 +17,34 @@ public static class StandardScripts
             PayToWitTemplate.Instance
         };
 
-        public static bool IsStandardTransaction(Transaction tx)
+        public static bool IsStandardTransaction(Transaction tx, Network network = null)
```

Comment:
Why default to Main rather than throw an exception?
also you could have `Network network = Network.Main` in the method signature.

Reply:
This is mostly to satisfy NBitcoin tests which are geared towards Network.Main